hera_mc/geo_handling.py

```diff
@@ -308,75 +308,71 @@ def get_ants_installed_since(self, query_date, station_types_to_check='all'):
         station_types_to_check:  list of stations types to limit check
         """
 
-        self.get_station_types(add_stations=False)
+        station_types_to_check = self.parse_station_types_to_check(station_types_to_check, add_stations=False)
         dt = query_date.gps
         found_stations = []
         for a in self.session.query(geo_location.GeoLocation).filter(
                 geo_location.GeoLocation.created_gpstime >= dt):
-            if (station_types_to_check == 'all' or
-                    self.flipped_station_types[a.station_type_name] in station_types_to_check):
+            if a.station_type_name.lower() in station_types_to_check:
                 found_stations.append(a.station_name)
         return found_stations
 
-    def plot_stations(self, stations_to_plot_list, query_date, state_args, testing=False):
+    def get_antenna_label(self, label_to_show, stn, query_date):
+        labeling = None
+        if label_to_show == 'name':
+            labeling = stn.station_name
```

Comment:
This could be:

```python
if label_to_show == 'name':
   return stn.station_name
```

And then the rest of the function could lose a level of indentation. It's generally a good thing to arrange control flow to minimize the levels of indentation (<=> levels of nested logic) that are needed. In general, in this function the `labeling` variable could be ditched in favor of just returning a value.